packages/editor/src/components/inserter/index.js

```diff
@@ -103,8 +103,13 @@ class Inserter extends Component {
 export default compose( [
 	withSelect( ( select, { rootClientId } ) => {
 		const {
-			getEditedPostAttribute,
 			hasInserterItems,
+		} = select( 'core/block-editor' );
+
+		// The title should be removed from the inserter
+		// or replaced by a prop passed to the inserter.
```

Comment:
Ya this is used when the `isMobile` is true for the Popover.  So is the idea here it shouldn't be tightly coupled to a _post_ editor title?  Maybe there should be a generic block-editor title that defaults to the post title if present?

Reply:
👍